frontend/appflowy_flutter/lib/workspace/presentation/home/menu/menu_user.dart

```diff
@@ -60,6 +81,50 @@ class MenuUser extends StatelessWidget {
     );
   }
 
+  Color _generateRandomNameColor(String name) {
```

Comment:
If it's truly necessary to generate the random color by name, just separate these functions into a util file.